src/resources/elements/questionMark/questionMark.scss

```diff
@@ -13,6 +13,5 @@
     align-items: center;
     height: 100%;
     position: relative;
```

Comment:
I think you can remove this line too

Reply:
Removed.
There is some other small issue I've noticed- On tiny displays, the labels have a `margin-bottom` of 5px, whereas the questionMark does not. It causes the questionMark not to be inlined with the label.

<img width="307" alt="Screenshot 2021-07-10 at 10 49 46" src="https://user-images.githubusercontent.com/2517870/125157718-cbfcec00-e16c-11eb-8658-f65a06fff205.png">

should I open a card for that?

Reply:
there is already a card on that, thanks

Reply:
Hmm, wait, this isn't this essentially the problem this this PR is supposed to be fixing?  Or perhaps your code change has actually exacerbated this one?

lets go ahead and fix it here.